src/App.vue

```diff
@@ -0,0 +1,64 @@
+      alt="user avatar"
+      class="header-avatar"
+    />
+  </header>
```

Comment:
The header contains a few elements, so it looks okay in this file. But if the code for this part is larger, it'll increase code complexity.

Maybe it's better to create a separate components for it? It can be placed in components -> layout folder

Reply:
Thank you for the idea, I remember your code style and it is a good idea to move some code to a separate layouts.
In my case, I think I can leave my current solution, when I will need to divide this block into components, I will do that.

Thank you for the idea again. 

If you don't mind, I will resolve this conversation